docs/_tutorials/phpqa-tutorial.md

```diff
@@ -30,6 +30,17 @@ Start PHPTesting!!!
 
 ## How to use it?
 
+### Create a git repository
+
+The `phpqa` tool expects you to be operating in the context of a `git`
+repository. So before continuing, you'll want to decide where you'll be storing
+your test files, and create a `git` repo in that directory.
```

Comment:
Sorry, I should have been more specific. This is related to the `phpqa generate` command, not the `phpqa run` command.